### DISCUSSION TITLE
How to steal your neighbors' thermometers: https://github.com/hdtodd/DNT

### DISCUSSION THREAD

**Discussion by hdtodd** [merbanan/rtl_433#2395] [Show and tell]
Well, ok, not "steal".  More like borrow, as in "may I borrow a cup of WiFi?"

I've written a small Python3 program to display neighboring thermometers using rtl_433 to monitor the ISM band.  It's available at [https://github.com/hdtodd/DNT](https://github.com/hdtodd/DNT).  It requires rtl_433 to be operating on a computer (perhaps the same computer) on your local-area network.  It functions on Raspbian and MacOSX (at least).

The README guides through the installation and configuration of rtl_433 as a monitoring service and of the DNT.py program.

David Todd

**Comment by zuckschwerdt** [comment]
Interesting to see a do-one-thing-well frontend to rtl_433 data.
We should really have a showcase or built-with-rtl_433 page in the wiki.
Start one if you like!

**Comment by hdtodd** [comment]
I like that idea!  I couldn't see a good way to share this code, but the 
"built-with rtl_433" idea is a great one.  I'll get one started tomorrow.

It occurs to me that my "snr" program might fit into the same category 
if I were to rework it to monitor the live stream. Interesting exercise 
for me to consider.  I'll give that some thought, too.


David

**Comment by hdtodd** [comment]
Done.  I added it to the rtl_433 Wiki page list. I haven't done one of 
these Wiki pages before. Please check and let me know if this format is OK.

David

**Reply by zuckschwerdt** [reply]
Looks good. You can also link that page from the wiki "home" page, e.g. under a new section "More".

**Comment by hdtodd** [comment]
I wasn't sure quite what you were suggesting, but I tried something that 
might attract interest in a slightly different way.

David
